messages/4.0.0-rc.1.txt

```diff
@@ -0,0 +1,64 @@
+SublimeLinter 4.0.0 Pre-Release
+===============================
+
+New features
+-------------------------------
+
+- Hover over an error to view the message.
+- Hover over the gutter to view all error messages for that line.
+- The statusbar now shows error (E) and warning (W) counts if there are any.
+- The statusbar shows the error for your cursor position instead of all errors
+  for that line.
+- A new "styles" setting to configure error highlighting.
+- Each linter setting now accepts such a "styles" setting, so colors and
+  highlight style can be differentiated per linter.
+- Styles also have a "priority" which decides what color to show in the gutter.
+- Gutter themes can now be set using just their name.
+- Generated "(SL)" color schemes are now hidden.
+
+On Sublime Text 3 builds > 3149 (dev builds):
+- No "(SL)" color scheme files are generated.
+- Select a color by specifying "region.<colorish>" in the scope option. Colors:
+  redish, orangish, yellowish, greenish, bluish, purplish, pinkish
+- It's also possible to use any other color by specifying any scope that's in
+  your color scheme (e.g. "entity.name")
+
+
+Changes to commands and menus
+-------------------------------
+
+The context menu now only list commands relevant for the view context.
+The tools menu now only lists the clear_cache command.
+Commands are now named sublime_linter_* to comply with naming conventions.
+
+Commands to change settings or toggle linters have been removed.
```

Comment:
What about the situation when you want to have a linter disabled by default but enable it for certain projects only?

Reply:
Sure, nothing about the enabled/disabled settings has changed, just the command has been removed. You can enable linters in project settings or in sublimelinterrc files.

Reply:
Right, didn't read this correctly. All good then.